data/map.txt

```diff
@@ -729,7 +730,62 @@ system Remnant
 		distance 1930
 		period 1598.78
 
-
+system Successor
+	pos -9820.108 -10035.952
+	habitable 955.249
+	belt 1535 9
+	belt 2444
+	haze star/void_scar
+	link Remnant
+	asteroids "small rock" 25 2.365
+	asteroids "medium rock" 2 2.398
+	asteroids "large rock" 11 1.685
+	asteroids "small metal" 42 2.981
+	asteroids "medium metal" 72 1.782
+	asteroids "large metal" 31 2.981
+	minables silver 20 2.491
+	minables uranium 20 3.848
+	object
+		sprite star/k5-old
+		distance 7.754
+		period 185.48
+		offset 180
+	object
+		sprite planet/ice5-b
+		distance 623
+		period 201.249
+	object
+		sprite planet/rock4
+		distance 1023
+		period 423.463
+	object Successor
+		sprite planet/water0
+		distance 1507
+		period 757.132
+	object
+		sprite planet/gas12-b
+		distance 2419
+		period 1539.769
+		object
+			sprite planet/rock0
+			distance 310
+			period 206.321
+	object
+		sprite planet/gas0-b
+		distance 5521
+		period 5309.192
+	object
+		sprite planet/gas7-r
+		distance 7524
+		period 8446.464
+		object
+			sprite planet/europa
+			distance 408
+			period 389.854
+		object
+			sprite planet/rock14-b
+			distance 550
+			period 337.568
```

Comment:
We usually copy the relevant capital/home system. So, here it'd be something like:
```suggestion
system Successor
	pos -9820.108 -10035.952
	arrival 3450
	habitable 3450
	belt 1179 3
	haze _menu/haze-67
	link Remnant
	asteroids "large metal" 10 3.062
	object
		sprite star/m-giant
		period 10
	object
		sprite planet/gas6-b
		distance 1311
		period 323.262
	object
		distance 2153
		period 680
		object
			sprite planet/ice8
			distance 120
			period 140
		object Successor
			sprite planet/dust0-b
			distance 120
			period 140
			offset 180
	object
		sprite planet/browndwarf-t
		distance 2804
		period 1011.154
	object
		sprite planet/neptune
		distance 4811
		period 2272.498
		object
			sprite planet/miranda
			distance 238
			period 173.691
	object
		sprite planet/jupiter
		distance 9120
		period 5931.202
		object
			sprite planet/desert4-b
			distance 263
			period 201.765
	object
		sprite planet/browndwarf-y
		distance 11064
		period 7925.354
		object
			sprite planet/rock7-b
			distance 293
			period 237.254
		object
			sprite planet/oberon-b
			distance 398
			period 489.011
```